protocol/messenger_backup.go

```diff
@@ -152,6 +152,38 @@ func (m *Messenger) BackupData(ctx context.Context) (uint64, error) {
 
 	}
 
+	cs, err = m.communitiesManager.DeletedCommunities()
+	if err != nil {
+		return 0, err
+	}
+	for _, c := range cs {
+		syncMessage, err := c.ToSyncCommunityProtobuf(clock)
+		if err != nil {
+			return 0, err
+		}
+
+		backupMessage := &protobuf.Backup{
+			Communities: []*protobuf.SyncCommunity{syncMessage},
+		}
+
+		encodedMessage, err := proto.Marshal(backupMessage)
+		if err != nil {
+			return 0, err
+		}
+
+		_, err = m.dispatchMessage(ctx, common.RawMessage{
+			LocalChatID:         chat.ID,
+			Payload:             encodedMessage,
+			SkipEncryption:      true,
+			SendOnPersonalTopic: true,
+			MessageType:         protobuf.ApplicationMetadataMessage_BACKUP,
+		})
+		if err != nil {
+			return 0, err
+		}
+
+	}
```

Comment:
Do you think that this could be wrapped into a function? I notice that it seems to be identical to the code at line 126.

Reply:
sure, I'm not even sure we need this code at all :D on it